Lesson2/webpack.config.js

```diff
@@ -0,0 +1,56 @@
+const path = require('path');
+const HTMLWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
+
+
+module.exports = {
+    mode: 'development',
+    entry: {
+        main: './js/main.js'
+    },
+    output: {
+        filename: '[name].[contenthash].js',
+        path: path.resolve(__dirname, 'dist'),
+        environment: {
+            module: true,
+            dynamicImport: true,
+        },
+    },
+    plugins: [
+        new HTMLWebpackPlugin({
+            template: 'index.html'
+        }),
+        new CleanWebpackPlugin(),
+        new MiniCssExtractPlugin({
+            filename: '[name].[contenthash].css'
+        }),
+
+        new BundleAnalyzerPlugin({
+            // Port that will be used by in `server` mode to start HTTP server. 
+            analyzerPort: 4000,
+        })
```

Comment:
BundleAnalyzerPlugin нужен только для анализа сборки, так что предлагаю из конечного варианта проекта его убрать
```suggestion
        new BundleAnalyzerPlugin({
            // Port that will be used by in `server` mode to start HTTP server. 
            analyzerPort: 4000,
        })
```

Reply:
ну и его импорт тоже